notices.json

```diff
@@ -4,12 +4,12 @@
     "conditions": {
       "audience": "all",
       "clientType": "desktop",
-      "desktopVersion": ["<4.6"]
+      "desktopVersion": ["<4.7"]
     },
     "localizedMessages": {
       "en": {
         "title": "New desktop version available",
-        "description": "Upgrade now to access custom desktop notification sounds, Russian spell check, and an improved user experience using multiple workspaces.",
+        "description": "Upgrade now to access Electron BrowserView, an underlying architecture change that improves performance and offers snappier interactions (i.e., less lag), lower CPU usage, and faster launch times.",
```

Comment:
We can probably cut this part of the sentence: "Electron BrowserView, an underlying architecture change that "

From the end user perspective they care about what it means to them